instrumentation/log4j/log4j-appender-2.17/javaagent/README.md

```diff
@@ -1,10 +1,10 @@
 # Settings for the Log4j Appender instrumentation
 
-| System property                                                                    | Type    | Default | Description                                                                                           |
-| ---------------------------------------------------------------------------------- | ------- | ------- | ----------------------------------------------------------------------------------------------------- |
-| `otel.instrumentation.log4j-appender.experimental-log-attributes`                  | Boolean | `false` | Enable the capture of experimental span attributes `thread.name` and `thread.id`.                     |
-| `otel.instrumentation.log4j-appender.experimental.capture-map-message-attributes`  | Boolean | `false` | Enable the capture of `MapMessage` attributes.                                                        |
-| `otel.instrumentation.log4j-appender.experimental.capture-marker-attribute`        | Boolean | `false` | Enable the capture of Log4j markers as attributes.                                                    |
-| `otel.instrumentation.log4j-appender.experimental.capture-context-data-attributes` | String  |         | List of context data attributes to capture. Use the wildcard character `*` to capture all attributes. |
+| System property                                                                    | Type    | Default | Description                                                                                                           |
```

Comment:
The white-space changes are from running the markdown formatting commands manually from #8968 (since it isn't automated yet). Not sure why it's formatting differently but this new format seems ok